src/Action/ResetAction.php

```diff
@@ -97,6 +97,9 @@ public function __construct(
         $this->tokenTtl = $tokenTtl;
     }
 
+    /**
+     * @param string $token
```

Comment:
Why not using php type instead ?

Reply:
It will be done on another Pr. Didnt want to break bc here (or not by a lot)